proto/v1/auth.proto

```
@@ -0,0 +1,176 @@
+syntax = "proto3";
+
+package arco.v1;
+
+import "google/protobuf/timestamp.proto";
+import "buf/validate/validate.proto";
```

Buf (1.54.0)

import "buf/validate/validate.proto": file does not exist

(COMPILE)